File: ncd/src/main/java/org/intelehealth/ncd/tabs/HypertensionScreeningFragment.java
package org.intelehealth.ncd.tabs;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import org.intelehealth.ekalarogya.databinding.LayoutSearchPatientCategoryBinding;

public class HypertensionScreeningFragment extends Fragment {

    private LayoutSearchPatientCategoryBinding binding;

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        binding = LayoutSearchPatientCategoryBinding.inflate(getLayoutInflater());
        return binding.getRoot();
    }
}

File: ncd/src/main/java/org/intelehealth/ncd/search/tabs/HypertensionScreeningFragment.kt
package org.intelehealth.ncd.search.tabs

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import org.intelehealth.ncd.databinding.LayoutSearchPatientCategoryBinding

class HypertensionScreeningFragment : Fragment() {

    private var binding: LayoutSearchPatientCategoryBinding? = null

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = LayoutSearchPatientCategoryBinding.inflate(layoutInflater)
        return binding!!.root
    }

}
